incorrect command to add labels to node for project wide node selector (#24865)
openshift/openshift-docs: nodes/scheduling/nodes-scheduler-node-selectors.adoc

A _node selector_ specifies a map of key-value pairs. The rules are defined using custom labels on nodes and selectors specified in pods.

> You can use specific node selectors to place specific pods on specific nodes, project node selectors to place new pods in a project on specific nodes in that project, or  default cluster-wide node selectors to place new pods on specific nodes anywhere in the cluster.

Reviewer: ```suggestion
You can use specific node selectors to place specific pods on specific nodes, project node selectors to place new pods in a project on specific nodes in that project, or default cluster-wide node selectors to place new pods on specific nodes anywhere in the cluster.
```

For the pod to be eligible to run on a node, the pod must have the same key-value node selector as the label on the node. 

> [NOTE]

Reviewer: ```suggestion
[IMPORTANT]
```

====

> If you are using node selectors and node affinity in the same pod configuration, note the following:

Reviewer: ```suggestion
If you are using node selectors and node affinity in the same pod configuration, the following rules control pod placement onto nodes:
```
Or similar. You need a noun after "following" at least.
